supervisor/shared/usb/usb.c

```diff
@@ -225,13 +225,17 @@ void usb_background_schedule(void) {
 }
 
 void usb_irq_handler(int instance) {
+    #if CFG_TUSB_MCU != OPT_MCU_RP2040
```

Comment:
Note: we can entirely skip usb_irq_handler() and simply call usb_background_schedule() in the _usb_irq_wrapper() in above rpi/supervisor/usb.c if you find this ifdef not appropriate. Though I think this change may make more sense for future maintaining should we want to add more code to this function